pom.xml

```diff
@@ -19,16 +19,18 @@
         <joda-time.version>2.7</joda-time.version>
         <logback.core.version>1.1.2</logback.core.version>
         <dozer.version>5.5.1</dozer.version>
+        <orika.version>1.5.0</orika.version>
 
         <longpolling.version>2.0.13</longpolling.version>
-        <uvms.common.version>2.0.17</uvms.common.version>
+        <uvms.common.version>2.0.18-SNAPSHOT</uvms.common.version>
```

Comment:
Don't like to be dependent on non exchange snapshot. 
- its not clear what is tested 
- all non exchange snapshots has to be manually removed to be able to release


usm4uvms is currently in 3.0.6-SNAPSHOT version i dev branch